docs/pages/sass.md

```diff
@@ -100,17 +100,50 @@ Each component has an **export mixin** which prints out the CSS for that compone
 @include foundation-everything;
 ```
 
-Our [starter projects](starter-projects.html) include the full list of imports, making it easy to comment out the components you don't need.
+Our [starter projects](starter-projects.html) include the full list of imports, making it easy to comment out the components you don't need. A full list is also included below. 
 
 ```scss
 @import 'foundation';
 
```

Comment:
`foundation` includes all the following files. So we could split the code block here and add a `Or`.